src/env.h

```diff
@@ -67,11 +67,18 @@ struct HasMain {
   enum Bool { Yes, No };
 };
 
+enum PackageMode {
+  NONE,
+  CJS,
+  ESM
+};
```

Comment:
I'd stick with, er, sticking it in a struct.  The enumerations end up in the `node::` scope now.  It also makes usage a little more self-explanatory (`PackageMode::NONE` instead of just `NONE`.  None what?)